src/Entity/ReferentManagedUsersMessage.php

```diff
@@ -252,4 +260,9 @@ public function setAgeMaximum(?int $ageMaximum): void
     {
         $this->ageMaximum = $ageMaximum;
     }
+
+    public function includeCP(): bool
```

Comment:
what about `getIncludeCitizenProject`? `includeCP` it is not clear if it's one getter or setter..